iquip/apps/monitor.py

```diff
@@ -209,6 +211,140 @@ def run(self):
             logger.exception("Failed to set the level of the target TTL channel.")
 
 
+class DACControllerWidget(QWidget):
+    """Single DAC channel controller widget.
+    
+    Attributes:
+        slider: Slider for setting the voltage.
+        voltageSpinBox: Spin box for setting and showing the voltage in slider.
+        setButton: Button for applying the voltage in practice.
```

Comment:
How about just `spinbox` and `button`, like `slider`, since there is no other spin box or button.